fun/module.py

```diff
@@ -811,7 +811,7 @@ def get_hyperpet_frames(avatar: Image.Image) -> List[Image.Image]:
 
         for i in range(6):
             img = "%02d" % (i + 1)
-            deform_hue = random.randint(0, 99) ** (i + 1) // 100**i / 100
+            deform_hue = random.randint(0, 99) ** (i + 1) // 100 ** i / 100
```

Comment:
Now there are two unrelated changes in this file, probably caused by mismatched black version. We should do something about synchronization of config of all repos, but I'm not sure what would be a proper solution.

Reply:
I know, but I think without this changes Black will not be happy about the PR.

Reply:
You can pass `git commit --no-verify` to skip those checks.

Reply:
Should I add this as separate commit? 🤔